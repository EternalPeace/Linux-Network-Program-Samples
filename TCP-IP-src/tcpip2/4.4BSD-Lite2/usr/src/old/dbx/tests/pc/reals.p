(*
 * Test of reals.
 *)

program realtest (input, output);
var
    x : real;
    y : real;

function f (x : real) : real;
begin
    f := 3.14*x;
end;

begin
    y := 3.0;
    x := f(y);
end.
