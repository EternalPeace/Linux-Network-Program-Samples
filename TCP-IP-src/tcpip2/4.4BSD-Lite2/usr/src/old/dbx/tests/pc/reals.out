reading symbolic information ...
7 
8.0 
[1] stop in program
[1] stopped in program at line 15
   15   begin
stopped in program at line 16
   16       y := 3.0;
stopped in program at line 17
   17       x := f(y);
3.0 
5.0 
10.5 
[3] stop in reals.f
[3] stopped in reals.f at line 11
   11   begin
reals.f(x = 3.0), line 11 in "reals.p"
program(), line 17 in "reals.p"
main(0x1, 0x7fffe7e8, 0x7fffe7f0) at 0xad

execution completed

